openwisp_controller/connection/tests/test_connection_notification.py

```diff
@@ -0,0 +1,124 @@
+from django.apps.registry import apps
```

Comment:
Can you rename this file `test_notifications.py` please? The fact that we're dealing with `connecton` is implicit from the path of the file.